reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md

```diff
@@ -606,8 +606,18 @@ Download [Raspbian Stretch](https://www.raspberrypi.org/downloads/raspbian/) and
 ### Installation - Raspbian
 
 ```sh
-# Install prerequisites
-sudo apt-get install libunwind8
+###################################
+# Prerequisites
+
+# Update package lists
+sudo apt-get update
+
+# Install libunwind8 and libssl1.0
+# Regex is used to ensure that we do not install libssl1.0-dev, as it is a variant that is not required
+sudo apt-get install '^libssl1.0(.?.?.?|.*([^v]|[^e]v|[^d]ev|[^-]dev))$' libunwind8 -y
```

Comment:
why do we need the regex at all?

Reply:
Hey Travis, I explored the option of just running:
`sudo apt-get install libssl1.0 libunwind8`
but apt-get inherently uses regex when it does not find an exact version match. So in this case, _libssl1.0_ matched to:

- libssl1.0.0
- libssl1.0.2, and
- libssl1.0-dev

libssl1.0.0 and 1.0.2 essentially are the same thing, and they are what we are trying to ensure is installed as a prerequisite. libssl1.0-dev is a different package that is not needed. In plain English, the regex expression is saying "give me the latest version of libssl1.0.x, but do not install libssl1.0-dev"

Hope that helps clarify. Thanks!

Reply:
wouldn't this work better?
```suggestion
sudo apt-get install '^libssl1.0.[0-9]$' libunwind8 -y
```

Reply:
This looks reasonable to me. Let me test it out on my Pi4 to make sure there's no issues with Raspbian Buster and I'll report back.

Reply:
OK, your suggestion worked well on Raspbian Buster. See screenshot:
<img width="329" alt="2019-07-10-213636_3840x2160_scrot_cropped" src="https://user-images.githubusercontent.com/11204406/61018254-bf661300-a35b-11e9-9c52-71ace1f2e54f.png">